src/EnergyPlus/SizingManager.cc

```diff
@@ -4477,7 +4480,7 @@ void ReportSysSizing(EnergyPlusData &state,
     if (state.dataSizingManager->ReportSysSizingMyOneTimeFlag) {
         print(state.files.eio,
               "{}\n",
-              "! <System Sizing Information>, System Name, Load Type, Peak Load Kind, User Design Capacity, Calc Des Air "
+              "! <System Sizing Information>, System Name, Load Type, Peak Load Kind, User Design Capacity [W], Calc Des Air "
```

Comment:
Added missing units to the `User Design Capacity [W]`. Now the unit appears in the `eio` and `html` table outputs. Without this unit in the table header, units conversion will not be performed correctly if user's preference in `html` tabular output is `InchPound`.

Reply:
Got it.  This looks like something we should refactor later.  Magically hiding the units in the string is unnecessary -- the units should just be an enum passed as a separate argument.  This is reminiscent of the setup output variable refactor.  But that's for a later date.